### PR TITLE
0.9.6 update

### PR DESCRIPTION
**Pull request by dgarcia360**
- updated 0.9.6 schemas #297 
- added new search commands #298 
- refactored closing aggregate transaction #299
- added missing unit tests (validators, resolves, transaction signature).

### CODE REVIEW

File: src/commands/transaction/accountaddressrestriction.ts [hunk 1]
```diff
@@ -69,14 +69,14 @@ export default class extends AnnounceTransactionsCommand {
...
-        const address = await new AddressAliasResolver().resolve(
+        const address = await new UnresolvedAddressResolver().resolve(
             options,
             undefined,
             'Enter the recipient address (or @alias):',
             'recipientAddress',
         );
         const maxFee = await new MaxFeeResolver().resolve(options);
-        const signerMultisigInfo = await this.getSignerMultisigInfo(options);
+        const signerMultisig = await this.getsignerMultisig(options);
```
Comment (rg911) [inline src/commands/transaction/accountaddressrestriction.ts:79]: camel case?

**Comment by dgarcia360** [comment]
Coveralls decreases because the PR includes some files that were not tested before (see commands and services folder) https://coveralls.io/builds/31655947

**Comment by dgarcia360** [comment]
blocked, pending to test changes against the latest REST.

**Comment by dgarcia360** [comment]
@fboucquez @rg911 We could start the review. If there are some issues after testing the new commands against a 0.9.6.1 rest instance, I'll submit an additional PR before the release of the CLI.

**Pull request merged**